src/datasources/balances-api/coingecko-api.service.ts

```diff
@@ -108,33 +116,49 @@ export class CoingeckoApi implements IPricesApi {
    * @param args.chainName Coingecko's name for the chain (see configuration)
    * @param args.tokenAddresses Array of token addresses which prices are being retrieved
    * @param args.fiatCode
-   * @returns Array of {@link AssetPrice}
+   * @returns Array of {@link CoingeckoAssetPrice}
    */
   async getTokenPrices(args: {
-    chainName: string;
+    chainId: string;
```

Comment:
This class is receiving a generic `chainId` instead of `chainName`. `chainName` is Coingecko-specific so I think it makes sense to hide this provider's implementation detail from the caller.